diff -U 1 /usr/include/glib-2.0/girepository/gicallableinfo.h /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/gicallableinfo.h
--- /usr/include/glib-2.0/girepository/gicallableinfo.h	2024-03-07 22:35:05.000000000 +0100
+++ /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/gicallableinfo.h	2026-07-01 20:02:38.723168210 +0200
@@ -118,2 +118,14 @@
 
+GI_AVAILABLE_IN_2_84
+GICallableInfo        *gi_callable_info_get_async_function (GICallableInfo *info);
+
+GI_AVAILABLE_IN_2_84
+GICallableInfo        *gi_callable_info_get_sync_function (GICallableInfo *info);
+
+GI_AVAILABLE_IN_2_84
+GICallableInfo        *gi_callable_info_get_finish_function (GICallableInfo *info);
+
+GI_AVAILABLE_IN_2_84
+gboolean               gi_callable_info_is_async (GICallableInfo *info);
+
 G_END_DECLS
diff -U 1 /usr/include/glib-2.0/girepository/girepository.h /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/girepository.h
--- /usr/include/glib-2.0/girepository/girepository.h	2024-03-07 22:35:05.000000000 +0100
+++ /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/girepository.h	2026-07-01 20:02:38.726819398 +0200
@@ -75,3 +75,3 @@
   GI_REPOSITORY_LOAD_FLAG_LAZY = 1 << 0
-} GIRepositoryLoadFlags;
+} G_GNUC_FLAG_ENUM GIRepositoryLoadFlags;
 
@@ -244,2 +244,5 @@
 
+GI_AVAILABLE_IN_2_86
+GIRepository *gi_repository_dup_default (void);
+
 #include <girepository/girepository-autocleanups.h>

