diff -U 1 /usr/include/glib-2.0/girepository/gicallableinfo.h /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/gicallableinfo.h
--- /usr/include/glib-2.0/girepository/gicallableinfo.h	2024-03-07 22:35:05.000000000 +0100
+++ /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/gicallableinfo.h	2026-07-01 20:02:38.723168210 +0200
@@ -118,2 +118,14 @@
 
+GI_AVAILABLE_IN_2_84
+GICallableInfo        *gi_callable_info_get_async_function (GICallableInfo *info);
+
+GI_AVAILABLE_IN_2_84
+GICallableInfo        *gi_callable_info_get_sync_function (GICallableInfo *info);
+
+GI_AVAILABLE_IN_2_84
+GICallableInfo        *gi_callable_info_get_finish_function (GICallableInfo *info);
+
+GI_AVAILABLE_IN_2_84
+gboolean               gi_callable_info_is_async (GICallableInfo *info);
+
 G_END_DECLS
diff -U 1 /usr/include/glib-2.0/girepository/girepository.h /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/girepository.h
--- /usr/include/glib-2.0/girepository/girepository.h	2024-03-07 22:35:05.000000000 +0100
+++ /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/girepository.h	2026-07-01 20:02:38.726819398 +0200
@@ -75,3 +75,3 @@
   GI_REPOSITORY_LOAD_FLAG_LAZY = 1 << 0
-} GIRepositoryLoadFlags;
+} G_GNUC_FLAG_ENUM GIRepositoryLoadFlags;
 
@@ -244,2 +244,5 @@
 
+GI_AVAILABLE_IN_2_86
+GIRepository *gi_repository_dup_default (void);
+
 #include <girepository/girepository-autocleanups.h>
diff -U 1 /usr/include/glib-2.0/girepository/girffi.h /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/girffi.h
--- /usr/include/glib-2.0/girepository/girffi.h	2024-03-07 22:35:05.000000000 +0100
+++ /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/girffi.h	2026-07-01 20:02:38.724087859 +0200
@@ -121,4 +121,4 @@
 GI_AVAILABLE_IN_ALL
-void **    gi_callable_info_get_closure_native_address (GICallableInfo       *callable_info,
-                                                        ffi_closure          *closure);
+void *gi_callable_info_get_closure_native_address (GICallableInfo *callable_info,
+                                                   ffi_closure *closure);
 
diff -U 1 /usr/include/glib-2.0/girepository/gitypes.h /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/gitypes.h
--- /usr/include/glib-2.0/girepository/gitypes.h	2024-03-07 22:35:05.000000000 +0100
+++ /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/gitypes.h	2026-07-01 20:02:38.725001206 +0200
@@ -368,2 +368,3 @@
  * GIFieldInfoFlags:
+ * @GI_FIELD_INFO_FLAGS_NONE: no flags set (since: 2.86)
  * @GI_FIELD_IS_READABLE: field is readable.
@@ -378,5 +379,6 @@
 {
+  GI_FIELD_INFO_FLAGS_NONE GI_AVAILABLE_ENUMERATOR_IN_2_86 = 0,
   GI_FIELD_IS_READABLE = 1 << 0,
   GI_FIELD_IS_WRITABLE = 1 << 1
-} GIFieldInfoFlags;
+} G_GNUC_FLAG_ENUM GIFieldInfoFlags;
 
@@ -384,2 +386,3 @@
  * GIVFuncInfoFlags:
+ * @GI_VFUNC_INFO_FLAGS_NONE: no flags set (since: 2.86)
  * @GI_VFUNC_MUST_CHAIN_UP: chains up to the parent type
@@ -394,2 +397,3 @@
 {
+  GI_VFUNC_INFO_FLAGS_NONE GI_AVAILABLE_ENUMERATOR_IN_2_86 = 0,
   GI_VFUNC_MUST_CHAIN_UP     = 1 << 0,
@@ -397,3 +401,3 @@
   GI_VFUNC_MUST_NOT_OVERRIDE = 1 << 2,
-} GIVFuncInfoFlags;
+} G_GNUC_FLAG_ENUM GIVFuncInfoFlags;
 
@@ -401,2 +405,3 @@
  * GIFunctionInfoFlags:
+ * @GI_FUNCTION_INFO_FLAGS_NONE: no flags set (since: 2.86)
  * @GI_FUNCTION_IS_METHOD: is a method.
@@ -413,2 +418,3 @@
 {
+  GI_FUNCTION_INFO_FLAGS_NONE GI_AVAILABLE_ENUMERATOR_IN_2_86 = 0,
   GI_FUNCTION_IS_METHOD      = 1 << 0,
@@ -418,3 +424,4 @@
   GI_FUNCTION_WRAPS_VFUNC    = 1 << 4,
-} GIFunctionInfoFlags;
+  GI_FUNCTION_IS_ASYNC       = 1 << 5,
+} G_GNUC_FLAG_ENUM GIFunctionInfoFlags;
 
diff -U 1 /usr/include/glib-2.0/girepository/gi-visibility.h /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/gi-visibility.h
--- /usr/include/glib-2.0/girepository/gi-visibility.h	2026-01-28 18:53:07.000000000 +0100
+++ /n4800/DATEN/Programmierung/mit_GIT/Lazarus/Tutorial/GNOME/Package_Tools/include-C/Ubuntu_26.04/girepository/gi-visibility.h	2026-07-01 20:02:38.726432950 +0200
@@ -986 +986,137 @@
 #endif
+
+#if GLIB_VERSION_MIN_REQUIRED >= GLIB_VERSION_2_82
+#define GI_DEPRECATED_IN_2_82 GI_DEPRECATED
+#define GI_DEPRECATED_IN_2_82_FOR(f) GI_DEPRECATED_FOR (f)
+#define GI_DEPRECATED_MACRO_IN_2_82 GLIB_DEPRECATED_MACRO
+#define GI_DEPRECATED_MACRO_IN_2_82_FOR(f) GLIB_DEPRECATED_MACRO_FOR (f)
+#define GI_DEPRECATED_ENUMERATOR_IN_2_82 GLIB_DEPRECATED_ENUMERATOR
+#define GI_DEPRECATED_ENUMERATOR_IN_2_82_FOR(f) GLIB_DEPRECATED_ENUMERATOR_FOR (f)
+#define GI_DEPRECATED_TYPE_IN_2_82 GLIB_DEPRECATED_TYPE
+#define GI_DEPRECATED_TYPE_IN_2_82_FOR(f) GLIB_DEPRECATED_TYPE_FOR (f)
+#else
+#define GI_DEPRECATED_IN_2_82 _GI_EXTERN
+#define GI_DEPRECATED_IN_2_82_FOR(f) _GI_EXTERN
+#define GI_DEPRECATED_MACRO_IN_2_82
+#define GI_DEPRECATED_MACRO_IN_2_82_FOR(f)
+#define GI_DEPRECATED_ENUMERATOR_IN_2_82
+#define GI_DEPRECATED_ENUMERATOR_IN_2_82_FOR(f)
+#define GI_DEPRECATED_TYPE_IN_2_82
+#define GI_DEPRECATED_TYPE_IN_2_82_FOR(f)
+#endif
+
+#if GLIB_VERSION_MAX_ALLOWED < GLIB_VERSION_2_82
+#define GI_AVAILABLE_IN_2_82 GI_UNAVAILABLE (2, 82)
+#define GI_AVAILABLE_STATIC_INLINE_IN_2_82 GLIB_UNAVAILABLE_STATIC_INLINE (2, 82)
+#define GI_AVAILABLE_MACRO_IN_2_82 GLIB_UNAVAILABLE_MACRO (2, 82)
+#define GI_AVAILABLE_ENUMERATOR_IN_2_82 GLIB_UNAVAILABLE_ENUMERATOR (2, 82)
+#define GI_AVAILABLE_TYPE_IN_2_82 GLIB_UNAVAILABLE_TYPE (2, 82)
+#else
+#define GI_AVAILABLE_IN_2_82 _GI_EXTERN
+#define GI_AVAILABLE_STATIC_INLINE_IN_2_82
+#define GI_AVAILABLE_MACRO_IN_2_82
+#define GI_AVAILABLE_ENUMERATOR_IN_2_82
+#define GI_AVAILABLE_TYPE_IN_2_82
+#endif
+
+#if GLIB_VERSION_MIN_REQUIRED >= GLIB_VERSION_2_84
+#define GI_DEPRECATED_IN_2_84 GI_DEPRECATED
+#define GI_DEPRECATED_IN_2_84_FOR(f) GI_DEPRECATED_FOR (f)
+#define GI_DEPRECATED_MACRO_IN_2_84 GLIB_DEPRECATED_MACRO
+#define GI_DEPRECATED_MACRO_IN_2_84_FOR(f) GLIB_DEPRECATED_MACRO_FOR (f)
+#define GI_DEPRECATED_ENUMERATOR_IN_2_84 GLIB_DEPRECATED_ENUMERATOR
+#define GI_DEPRECATED_ENUMERATOR_IN_2_84_FOR(f) GLIB_DEPRECATED_ENUMERATOR_FOR (f)
+#define GI_DEPRECATED_TYPE_IN_2_84 GLIB_DEPRECATED_TYPE
+#define GI_DEPRECATED_TYPE_IN_2_84_FOR(f) GLIB_DEPRECATED_TYPE_FOR (f)
+#else
+#define GI_DEPRECATED_IN_2_84 _GI_EXTERN
+#define GI_DEPRECATED_IN_2_84_FOR(f) _GI_EXTERN
+#define GI_DEPRECATED_MACRO_IN_2_84
+#define GI_DEPRECATED_MACRO_IN_2_84_FOR(f)
+#define GI_DEPRECATED_ENUMERATOR_IN_2_84
+#define GI_DEPRECATED_ENUMERATOR_IN_2_84_FOR(f)
+#define GI_DEPRECATED_TYPE_IN_2_84
+#define GI_DEPRECATED_TYPE_IN_2_84_FOR(f)
+#endif
+
+#if GLIB_VERSION_MAX_ALLOWED < GLIB_VERSION_2_84
+#define GI_AVAILABLE_IN_2_84 GI_UNAVAILABLE (2, 84)
+#define GI_AVAILABLE_STATIC_INLINE_IN_2_84 GLIB_UNAVAILABLE_STATIC_INLINE (2, 84)
+#define GI_AVAILABLE_MACRO_IN_2_84 GLIB_UNAVAILABLE_MACRO (2, 84)
+#define GI_AVAILABLE_ENUMERATOR_IN_2_84 GLIB_UNAVAILABLE_ENUMERATOR (2, 84)
+#define GI_AVAILABLE_TYPE_IN_2_84 GLIB_UNAVAILABLE_TYPE (2, 84)
+#else
+#define GI_AVAILABLE_IN_2_84 _GI_EXTERN
+#define GI_AVAILABLE_STATIC_INLINE_IN_2_84
+#define GI_AVAILABLE_MACRO_IN_2_84
+#define GI_AVAILABLE_ENUMERATOR_IN_2_84
+#define GI_AVAILABLE_TYPE_IN_2_84
+#endif
+
+#if GLIB_VERSION_MIN_REQUIRED >= GLIB_VERSION_2_86
+#define GI_DEPRECATED_IN_2_86 GI_DEPRECATED
+#define GI_DEPRECATED_IN_2_86_FOR(f) GI_DEPRECATED_FOR (f)
+#define GI_DEPRECATED_MACRO_IN_2_86 GLIB_DEPRECATED_MACRO
+#define GI_DEPRECATED_MACRO_IN_2_86_FOR(f) GLIB_DEPRECATED_MACRO_FOR (f)
+#define GI_DEPRECATED_ENUMERATOR_IN_2_86 GLIB_DEPRECATED_ENUMERATOR
+#define GI_DEPRECATED_ENUMERATOR_IN_2_86_FOR(f) GLIB_DEPRECATED_ENUMERATOR_FOR (f)
+#define GI_DEPRECATED_TYPE_IN_2_86 GLIB_DEPRECATED_TYPE
+#define GI_DEPRECATED_TYPE_IN_2_86_FOR(f) GLIB_DEPRECATED_TYPE_FOR (f)
+#else
+#define GI_DEPRECATED_IN_2_86 _GI_EXTERN
+#define GI_DEPRECATED_IN_2_86_FOR(f) _GI_EXTERN
+#define GI_DEPRECATED_MACRO_IN_2_86
+#define GI_DEPRECATED_MACRO_IN_2_86_FOR(f)
+#define GI_DEPRECATED_ENUMERATOR_IN_2_86
+#define GI_DEPRECATED_ENUMERATOR_IN_2_86_FOR(f)
+#define GI_DEPRECATED_TYPE_IN_2_86
+#define GI_DEPRECATED_TYPE_IN_2_86_FOR(f)
+#endif
+
+#if GLIB_VERSION_MAX_ALLOWED < GLIB_VERSION_2_86
+#define GI_AVAILABLE_IN_2_86 GI_UNAVAILABLE (2, 86)
+#define GI_AVAILABLE_STATIC_INLINE_IN_2_86 GLIB_UNAVAILABLE_STATIC_INLINE (2, 86)
+#define GI_AVAILABLE_MACRO_IN_2_86 GLIB_UNAVAILABLE_MACRO (2, 86)
+#define GI_AVAILABLE_ENUMERATOR_IN_2_86 GLIB_UNAVAILABLE_ENUMERATOR (2, 86)
+#define GI_AVAILABLE_TYPE_IN_2_86 GLIB_UNAVAILABLE_TYPE (2, 86)
+#else
+#define GI_AVAILABLE_IN_2_86 _GI_EXTERN
+#define GI_AVAILABLE_STATIC_INLINE_IN_2_86
+#define GI_AVAILABLE_MACRO_IN_2_86
+#define GI_AVAILABLE_ENUMERATOR_IN_2_86
+#define GI_AVAILABLE_TYPE_IN_2_86
+#endif
+
+#if GLIB_VERSION_MIN_REQUIRED >= GLIB_VERSION_2_88
+#define GI_DEPRECATED_IN_2_88 GI_DEPRECATED
+#define GI_DEPRECATED_IN_2_88_FOR(f) GI_DEPRECATED_FOR (f)
+#define GI_DEPRECATED_MACRO_IN_2_88 GLIB_DEPRECATED_MACRO
+#define GI_DEPRECATED_MACRO_IN_2_88_FOR(f) GLIB_DEPRECATED_MACRO_FOR (f)
+#define GI_DEPRECATED_ENUMERATOR_IN_2_88 GLIB_DEPRECATED_ENUMERATOR
+#define GI_DEPRECATED_ENUMERATOR_IN_2_88_FOR(f) GLIB_DEPRECATED_ENUMERATOR_FOR (f)
+#define GI_DEPRECATED_TYPE_IN_2_88 GLIB_DEPRECATED_TYPE
+#define GI_DEPRECATED_TYPE_IN_2_88_FOR(f) GLIB_DEPRECATED_TYPE_FOR (f)
+#else
+#define GI_DEPRECATED_IN_2_88 _GI_EXTERN
+#define GI_DEPRECATED_IN_2_88_FOR(f) _GI_EXTERN
+#define GI_DEPRECATED_MACRO_IN_2_88
+#define GI_DEPRECATED_MACRO_IN_2_88_FOR(f)
+#define GI_DEPRECATED_ENUMERATOR_IN_2_88
+#define GI_DEPRECATED_ENUMERATOR_IN_2_88_FOR(f)
+#define GI_DEPRECATED_TYPE_IN_2_88
+#define GI_DEPRECATED_TYPE_IN_2_88_FOR(f)
+#endif
+
+#if GLIB_VERSION_MAX_ALLOWED < GLIB_VERSION_2_88
+#define GI_AVAILABLE_IN_2_88 GI_UNAVAILABLE (2, 88)
+#define GI_AVAILABLE_STATIC_INLINE_IN_2_88 GLIB_UNAVAILABLE_STATIC_INLINE (2, 88)
+#define GI_AVAILABLE_MACRO_IN_2_88 GLIB_UNAVAILABLE_MACRO (2, 88)
+#define GI_AVAILABLE_ENUMERATOR_IN_2_88 GLIB_UNAVAILABLE_ENUMERATOR (2, 88)
+#define GI_AVAILABLE_TYPE_IN_2_88 GLIB_UNAVAILABLE_TYPE (2, 88)
+#else
+#define GI_AVAILABLE_IN_2_88 _GI_EXTERN
+#define GI_AVAILABLE_STATIC_INLINE_IN_2_88
+#define GI_AVAILABLE_MACRO_IN_2_88
+#define GI_AVAILABLE_ENUMERATOR_IN_2_88
+#define GI_AVAILABLE_TYPE_IN_2_88
+#endif
